microservice/settings/src/test/java/com/cumulocity/microservice/settings/service/MicroserviceSettingsServiceTest.java

```diff
@@ -32,7 +34,7 @@
 
     @BeforeEach
     public void setUp() {
-        doReturn(new OptionsRepresentation()).when(contextService).callWithinContext(any(MicroserviceCredentials.class), any(Callable.class));
+        lenient().doReturn(new OptionsRepresentation()).when(contextService).callWithinContext(any(MicroserviceCredentials.class), any(Callable.class));
```

Comment:
`lenient()` usually means "I'm doing it wrong"

Reply:
I can move it to two existing tests. Since we first check if we are in context we never use "callWithinContext" in scope of the newly added test